interview-questions.md

```diff
@@ -8,36 +8,135 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 action: "GET" location: /students
 
+What is the name of the controller method that would be called by each route?
+the name of the controller method would be Index
+What action would each of the controller methods perform?
+the index action would be to list all of the items in a particular model.
+
 action: "POST" location: /students
+What is the name of the controller method that would be called by each route?
+the name of the controller method would be Create
+What action would each of the controller methods perform?
+it would create an new instance of students
 
 action: "GET" location: /students/new
 
+What is the name of the controller method that would be called by each route?
+the name of the controller method is New
+What action would each of the controller methods perform?
+the action of the controller method New would be to display a new form to the user
+
 action: "GET" location: /students/2
+What is the name of the controller method that would be called by each route?
+the controller method will be called Show
+What action would each of the controller methods perform?
+Show controller method will list one specific item in a particular model. In this instance I would like to list the 2nd students within the student model.
 
 action: "GET" location: /students/2/edit
+What is the name of the controller method that would be called by each route?
+the controller method will be called Edit
+What action would each of the controller methods perform?
+Show controller method will get data from one specific item in a particular model. In this instance I would like to get a form of the 2nd student within the student model.
+
 
 action: "PATCH" location: /students/2
 
+What is the name of the controller method that would be called by each route? the name of the controller method for this route will be Update.
+What action would each of the controller methods perform?
+Update controller method will bes used to modify information in the database. In this instance I want to modify information in the model for the 2nd student within the database. 
+
 action: "DELETE" location: /students/2
+What is the name of the controller method that would be called by each route? The name of the controller method will be Destroy
+What action would each of the controller methods perform?
+this controller method will be used to remove information from the database. In this case, I want to remove the data for my second student from the database. 
+
+Resources: https://github.com/learn-academy-2023-alpha/Syllabus/blob/main/rails/restful-routes-crud.md
 
 5. As a developer, you are making an application to manage your to do list. Create 10 user stories that will help you get your application started. Read more about [user stories](https://www.atlassian.com/agile/project-management/user-stories).
+
+🤠 As a user, I can see every tasks I have to accomplish on my to do list
```

Comment:
Wow!  You took it a step further and added acceptance criteria!  Awesome work on this problem and all of your interview answers! 👏